src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/incidents/QuestionnaireResourceIntTest.kt

```diff
@@ -43,12 +43,20 @@ class QuestionnaireResourceIntTest : IntegrationTestBase() {
           questionnaireAnswer(answer = "Q1A1: Yes", listSequence = 1, answerSequence = 1, nextQuestion = question2)
           questionnaireAnswer(answer = "Q1A2: No", listSequence = 2, answerSequence = 2, nextQuestion = question2)
         }
+        offenderRole("ABS")
+        offenderRole("ESC")
+        offenderRole("FIGHT")
+        offenderRole("PERP")
```

Comment:
I wonder if you could create a version of this that accepts a collection, so you can write `offenderRoles("ABS","ESC","FIGHT","PERP"). Might reduce a little noise.

Reply:
will try and tidy it up